analyses/cell-type-wilms-tumor-06/notebook-template/01-clustering.Rmd

```diff
@@ -0,0 +1,768 @@
+---
+title: "Evaluation of clustering parameters"
+author: maud-p
+date: '`r format(Sys.time(), "%B %d, %Y %H:%M:%S %Z")`'
+output:
+  html_document: 
+    toc: yes
+    toc_float: yes
+    code_folding: hide
+    highlight: pygments
+    df_print: kable
+params:
+  sample_name: ~
+  sample_path: ~
+  out_rds_path: ~
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+	echo = TRUE,
+	message = FALSE,
+	warnings = FALSE,
+	error = TRUE
+)
+```
+
+
+## Introduction
+
+This a template for an analysis notebook using RMarkdown.
+
+In this notebook, we will set up parameters in the seurat workflow [normalization --> reduction --> clustering] for one Wilms tumor sample (SCPCS000169) of the Wilms Tumor dataset (SCPCP000006). 
+
+This correspond to the step 2 of the proposed analysis: clustering of cells across a set of parameters for few samples
+
+- https://github.com/AlexsLemonade/OpenScPCA-analysis/discussions/635
+- https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/671
+
+In order to assess the clustering quality, we look into some marker genes, pathways enrichment and label transfer.
+
+
+
+## Sample metadata
+
+```{r paths2}
+## to be modified when rendered
+filedir <- filelist[i] 
+sample <- gsub(paste0(data_dir, "/"), "", filedir)
+
+metadata <- read.table(file.path(data_dir, "/single_cell_metadata.tsv"), sep = "\t", header=TRUE)
+
+metadata[metadata$scpca_sample_id == sample,]
+```
```

Comment:
This entire chunk can be deleted, since you will be using a different strategy to read in files, and we don't need this `metadata` data frame.